app/controllers/feedbacks_controller.rb

```diff
@@ -4,7 +4,7 @@
 class FeedbacksController < ApplicationController
   def create
     respond_to do |format|
-      if verify_recaptcha
+      if verify_recaptcha || params[:feedback_type] == 'Useful'
```

Comment:
@HanaBene ma zmysel posielat to useful do toho google formu? Ked budu pocty v GA tak asi ani nie, ze?

Reply:
nutne to nemusime mat na dvoch miestach, GA je v pohode

Reply:
@jsuchal tak davam to useful prec, nech sa do formu posiela len negativna/bug report